### ISSUE TITLE
Potential bug for the orbital-mask calculating in QH9 dataset

### ISSUE THREAD

**Issue by Franklalalala** [divelab/AIRS#20]
During the review of the source code of QHNet and QH9, I found a very strange setting.

That is, when one is trying to calculate the mask for H and He, the default orbital_mask is named with idx_1s_2s_2p and assigned with torch.tensor([0, 1, 3, 4, 5]), see [code](https://github.com/divelab/AIRS/blob/3f6c404b65171f8577913368c75040f2408f6b9b/OpenDFT/QHBench/QH9/datasets.py#L118).

However, when it comes to heavy atoms, it is initilized with torch.arange(self.full_orbitals), which do not have a gap in between, like torch.tensor([0, 1, 3, 4, 5]).

Similarly, there are other place in this repo, see [search results](https://github.com/search?q=repo%3Adivelab%2FAIRS%203%2C%204%2C%205&type=code)

I'm not sure whether this unusual setting could affect the performance. Probably not. But this setting is very strange.

**Comment by Franklalalala** [comment]
I have no questions right now!
